socrates/lib/site/views/schedule.jade

```diff
@@ -20,34 +20,66 @@ block content
         .panel-heading
           h4.panel-title Open Space Technology
         .panel-body.
-          is an approach for hosting meetings, conferences, corporate-style retreats and community summit events, 
-          focused on a specific and important purpose or task &ndash; but beginning without any formal agenda, beyond 
+          is an approach for hosting meetings, conferences, corporate-style retreats and community summit events,
+          focused on a specific and important purpose or task &ndash; but beginning without any formal agenda, beyond
           the overall purpose or theme.
         .panel-footer
           cite
             +extlink('http://en.wikipedia.org/wiki/Open_Space_Technology','Wikipedia')
       dl
-        dt Thursday evening
-        dd Opening: A #{''}
+        dt Thursday
+        dd.
+          The conference starts at 5pm on Thursday, August 27th 2015.
+          This is the time to get to know new faces or meet old friends, have
+          some drinks at the bar, explore the surrounding area etc.
+          T-Shirts are bing distributed and deposits are returned.
```

Comment:
Oh, this should be "being" I guess - could you just change it and commit the change? This will go into this pull request as well.